internal/tasks/tasks.go

```diff
@@ -164,213 +164,208 @@ func ExecuteTask(task Task) Task {
 		log.Fatal(err.Error())
 	}
 
-	if diagnostics.GetReleaseVersion() == diagnostics.DEV_VERSION {
```

Comment:
Will always run tasks now - no more blocking against dev

Reply:
Yeah, only made sense in the very beginning when we didn't have any way to isolate :P